### ISSUE TITLE
Not working But Ty for trying to keep up with the unstable build. 

### ISSUE THREAD

**Issue by Connorisgameing** [edbmods/EdBPrepareCarefully#228]
I want to say Ty Lot's of people and I love this mod<3.
(don't know how much it will help) but when I try to open it nothing happened so i used debug and got this. 
Debug:
RimWorld 1.0.1967 rev980
Verse.Log:Message(String, Boolean) (at C:\Dev\RimWorld\Assets\Scripts\Verse\Utility\Debug\Log\Log.cs:49)
RimWorld.VersionControl:LogVersionNumber() (at C:\Dev\RimWorld\Assets\Scripts\RimWorld\Utility\Version\VersionControl.cs:83)
Verse.Root:CheckGlobalInit() (at C:\Dev\RimWorld\Assets\Scripts\Verse\Global\Root\Root.cs:82)
Verse.Root:Start() (at C:\Dev\RimWorld\Assets\Scripts\Verse\Global\Root\Root.cs:38)
Verse.Root_Entry:Start() (at C:\Dev\RimWorld\Assets\Scripts\Verse\Global\Root\Root_Entry.cs:16)

Exception filling window for RimWorld.Page_ConfigureStartingPawns: System.InvalidOperationException: Comparison threw an exception. ---> System.MissingMethodException: Method not found: 'Verse.GenderUtility.GetLabel'.
  at EdB.PrepareCarefully.EquipmentRecord.get_Label () [0x00000] in <filename unknown>:0 
  at EdB.PrepareCarefully.EquipmentDatabase+<>c.<get_Animals>b__33_1 (EdB.PrepareCarefully.EquipmentRecord a, EdB.PrepareCarefully.EquipmentRecord b) [0x00000] in <filename unknown>:0 
  at System.Array.qsort[EquipmentRecord] (EdB.PrepareCarefully.EquipmentRecord[] array, Int32 low0, Int32 high0, System.Comparison`1 comparison) [0x00025] in /Users/builduser/buildslave/mono/build/mcs/class/corlib/System/Array.cs:1773 
  at System.Array.Sort[EquipmentRecord] (EdB.PrepareCarefully.EquipmentRecord[] array, Int32 length, System.Comparison`1 comparison) [0x00028] in /Users/builduser/buildslave/mono/build/mcs/class/corlib/System/Array.cs:1700 
  --- End of inner exception stack trace ---
at System.Array.Sort<EdB.PrepareCarefully.EquipmentRecord> (EdB.PrepareCarefully.EquipmentRecord[],int,System.Comparison`1<EdB.PrepareCarefully.EquipmentRecord>) <0x000cc>
at System.Collections.Generic.List`1<EdB.PrepareCarefully.EquipmentRecord>.Sort (System.Comparison`1<EdB.PrepareCarefully.EquipmentRecord>) <0x0003d>
at EdB.PrepareCarefully.EquipmentDatabase.get_Animals () <IL 0x00041, 0x000da>
at EdB.PrepareCarefully.PrepareCarefully.RandomPet (RimWorld.ScenPart_StartingAnimal) <IL 0x00093, 0x001d4>
at EdB.PrepareCarefully.PrepareCarefully.InitializeDefaultEquipment () <IL 0x001b8, 0x00612>
at EdB.PrepareCarefully.PrepareCarefully.Initialize () <IL 0x00029, 0x00050>
at EdB.PrepareCarefully.HarmonyPatches.DoWindowContentsPostfix (UnityEngine.Rect,RimWorld.Page_ConfigureStartingPawns) <IL 0x0006c, 0x00171>
at (wrapper dynamic-method) RimWorld.Page_ConfigureStartingPawns.DoWindowContents_Patch1 (object,UnityEngine.Rect) <IL 0x000be, 0x0052b>
at Verse.Window/<WindowOnGUI>c__AnonStorey0.<>m__0 (int) [0x00223] in C:\Dev\RimWorld\Assets\Scripts\Verse\UI\Windows\Window.cs:202

Verse.Log:Error(String, Boolean) (at C:\Dev\RimWorld\Assets\Scripts\Verse\Utility\Debug\Log\Log.cs:78)
Verse.<WindowOnGUI>c__AnonStorey0:<>m__0(Int32) (at C:\Dev\RimWorld\Assets\Scripts\Verse\UI\Windows\Window.cs:206)
UnityEngine.GUI:CallWindowDelegate(WindowFunction, Int32, Int32, GUISkin, Int32, Single, Single, GUIStyle) (at C:\buildslave\unity\build\Runtime\IMGUI\Managed\GUI.cs:1817)

**Comment by burningmime** [comment]
Did you pull the newest Assembly-CSharp.dll from RimWorld into the libraries folder, or are you using an old one? Every time you do a new build, you need to copy in the latest assembly from the game.

**Comment by Connorisgameing** [comment]
No, how would I go about doing that?

**Comment by Connorisgameing** [comment]
Yeah i had the new release 4 I got it when i posted this, but I re-downloaded and installed it. Now it's working, TY for the response gave me the thought of just re-downloading and installing. Probably got corrupted or something and couldn't find the right files.

Have a good one, I know I will trying to survive naked brutality. (with a decent pawn)
